check50/__main__.py

```diff
@@ -278,17 +284,25 @@ def main():
     global SLUG
     SLUG = args.slug
 
-
     if args.dev:
         args.offline = True
-        args.verbose = True
+        args.verbose = "info"
 
     if args.offline:
         args.local = True
 
-    if args.verbose:
-        # Show lib50 commands being run in verbose mode
-        logging.basicConfig(level=os.environ.get("CHECK50_LOGLEVEL", "INFO"))
+    if args.verbose == "info":
+        # Show lib50 commands being run
+        lib50_logger = logging.getLogger("lib50")
+        lib50_logger.setLevel(logging.INFO)
+        lib50_logger.addHandler(logging.StreamHandler(sys.stdout))
```

Comment:
I wonder if log output should be logged out to `stderr` instead. That way you can easily do 

```
check50 ... 2>check50.log
```
To get a file only containing the logged output which might be useful?

Reply:
Done